<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YWindow</class>
 <widget class="QMainWindow" name="YWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1201</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>牛览器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btnBackward">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/image/img/png-1639.png</normaloff>:/image/img/png-1639.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btnForward">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/image/img/png-1679.png</normaloff>:/image/img/png-1679.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btnRefresh">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/image/img/png-0651.png</normaloff>:/image/img/png-0651.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="objectAddress"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btnGO">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/image/img/png-0650.png</normaloff>:/image/img/png-0650.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QPushButton" name="btnFavorite">
      <property name="text">
       <string extracomment="收藏"/>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/image/img/png-1663.png</normaloff>:/image/img/png-1663.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QPushButton" name="FavoriteList">
      <property name="maximumSize">
       <size>
        <width>33</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>页</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="initialTab">
       <attribute name="title">
        <string>+</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionSet"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>功能</string>
    </property>
    <widget class="QMenu" name="menuHistory">
     <property name="title">
      <string>历史</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuFavorite">
     <property name="title">
      <string>收藏</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuTempFavorite">
     <property name="title">
      <string>当前收藏</string>
     </property>
    </widget>
    <addaction name="menuHistory"/>
    <addaction name="menuFavorite"/>
    <addaction name="menuTempFavorite"/>
   </widget>
   <widget class="QMenu" name="menuFavoriteDir">
    <property name="title">
     <string>收藏夹</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFunction"/>
   <addaction name="menuFavoriteDir"/>
  </widget>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/img/png-0011.png</normaloff>:/image/img/png-0011.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>界面退出</string>
   </property>
  </action>
  <action name="actionSet">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/img/png-0007.png</normaloff>:/image/img/png-0007.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置功能</string>
   </property>
  </action>
  <action name="actionffff">
   <property name="text">
    <string>ffff</string>
   </property>
  </action>
  <action name="action1111">
   <property name="text">
    <string>1111</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
